wiki/Gameplay/Input_device/Drum_controller/en.md

```diff
@@ -0,0 +1,8 @@
+# Drum Controller
+
+**Drum controllers** are input devices designed to work with taiko drum games. 
+For PS4 and Switch drum controllers, you will need to plug it into your computer and bind the correct keys in the osu! settings.
+For the Wii Tatacon, you will need to pair it to your computer first and enable Wiimote/Tatacon support in the osu! settings. 
+Most 3rd party drum controllers will also work with osu!, but support may vary. 
+
+<!-- this is a stub -->
```

Comment:
same deal as before; i wont be commenting on these anymore, but just know that all articles should have the comment placed at the top with some tags preceeding it.